src/lib/dom-api.html

```diff
@@ -172,10 +172,11 @@
         // when the need arises.  For now, the user must call
         // distributeContent(true), which updates insertion points manually
         // and forces distribution.
-       var fragContent = (node.nodeType === Node.DOCUMENT_FRAGMENT_NODE) &&
-            node.querySelector(CONTENT);
+        var fragContent = (node.nodeType === Node.DOCUMENT_FRAGMENT_NODE) &&
+          !node._noContent && Polymer.dom(node).querySelector(CONTENT);

```

Comment:
I was picturing the flag to be opt-in to the scoped querySelector cost, vs opt-out.  i.e. "tell us when you have a content in there"  as opposed to "tell us when you don't", but I guess it's not trivial for annotator to put the flag on, because it would need to be copied from `template.content` to the `root` fragment after `importNode`, is that right?